src/client/utils/throttle.js

```diff
@@ -0,0 +1,31 @@
+import nativeMethods from '../sandbox/native-methods';
+
+export default function throttle (func, ms) {
+    var isSuspend = false;
```

Comment:
I think we can completely remove throttling and `setCookie` service messages and
perform cookie synchronization with server only for cases from list above.